app/component/gallery/gallery-item/gallery-item.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+require('./_gallery-item.scss');
```

Comment:
Hey Nikko.  You are referencing this scss file but it doesn't exist so it is preventing webpack from building your app.